### DISCUSSION TITLE
Welcome to the new RDF4J discussion forum!

### DISCUSSION THREAD

**Discussion by abrokenjester** [eclipse-rdf4j/rdf4j#3089] [Announcements]
Hi everyone, 

To eventually replace our rdf4j-users Google group, we are trialling this new Github feature: [Github Discussions](https://docs.github.com/en/discussions). If you're here, you've found it!

Feel free to ask your questions about RDF4J here, or to propose new features, or even do a show and tell of some project involving RDF4J. Don't forget to upvote questions or answers you think are good or helpful, and by all means feel free to let us know what you think of this new way of providing community support!

Just a couple of housekeeping rules: 

 - We have a [code of conduct](https://github.com/eclipse/.github/blob/e3f5e7b272c5b4b0c8ffb6811cece7207942a413/CODE_OF_CONDUCT.md), please follow those guidelines  
  - Stick to the general topic, which is the Eclipse RDF4J framework
  - If you have a question or a problem, please make sure you include sufficient detail to make the problem reproducible: show a runnable sample of your code, or your query, and the exact error you're getting.  Please use the [markdown formatting](https://guides.github.com/features/mastering-markdown/) to your advantage, so that your post is easy to read

We have a couple of categories for discussions. These are currently pretty much the standard set, we may change that as we get more experience with the forum. Give it a go, let us know what you think!

**Comment by naturzukunft** [comment]
Hi, Github discussions can only be better than Google groups. I don't know it, but I love it already ;-)
